cmd/ledger/main.go

```diff
@@ -24,6 +25,7 @@ var (
 	compare       = app.Command("compare", "Compare two ledgers via their snapshots.")
 	snapshotPath1 = compare.Arg("snapshotPath1", "File path to first ledger snapshot.").Required().String()
 	snapshotPath2 = compare.Arg("snapshotPath2", "File path to second ledger snapshot.").Required().String()
+	destination   = compare.Flag("destination", "File path to snapshot comparison json results. Default is the current directory.").Short('d').String()
```

Comment:
It's not the `File path` that you expect here. You use this input as a folder and create a file in that. From that point of view, it would be better to call this flag `outputdir`

Reply:
Similarly, for the existing `args`, instead of `File path to first ledger snapshot` `first ledger snapshot dir` makes it clearer.